### PR TITLE
add Laravel 11 support

### PR DESCRIPTION
**Pull request by atmonshi**

### CODE REVIEW

**Comment by zepfietje** [comment]
Thanks for creating this PR @atmonshi.
This is my final dependency that doesn't support Laravel 11 yet.
Could you have a look and merge this PR, @GrahamCampbell? 🙂

**Comment by sebastiansulinski** [comment]
Any chance this could be actioned sometime soon?

**Comment by atmonshi** [comment]
I added the version `12.5` for laravel 11 in the readme table, not sure if it will be a major or minor version.
feel free to edit or request a change :)

**Comment by Oleksandr-Moik** [comment]
Hi there!

I sent a pull request to https://github.com/GrahamCampbell/Laravel-GitLab/pull/48 and noticed a difference in setup in the `tests.yml` configurations. 

@atmonshi, please, look and make similar changes 😉 (https://github.com/GrahamCampbell/Laravel-GitLab/pull/48/commits/5fabfa30a0e498041bbc27254d0d5e8b12d998c7)

File: README.md [hunk 1]
```diff
@@ -18,14 +18,15 @@ Laravel GitHub was created by, and is maintained by [Graham Campbell](https://gi
 
 This version requires [PHP](https://www.php.net/) 7.4-8.3 and supports [Laravel](https://laravel.com/) 8-10.
```
Comment (Oleksandr-Moik) [inline README.md]:
```suggestion
This version requires [PHP](https://www.php.net/) 7.4-8.3 and supports [Laravel](https://laravel.com/) 8-11.
```

**Comment by GrahamCampbell** [comment]
Thanks for getting started here, though I already have changes locally made in batch for all my packages, that I'm rolling out over the coming days.

**Pull request closed without merging**